filter/expand_json.go

```diff
@@ -10,11 +10,48 @@ import (
 	"github.com/jmespath/go-jmespath"
 )
 
+const help = `
+The filter copies the values of a set of JSON keys to corresponding record fields.
+It supports [JMESpath](https://jmespath.org/tutorial.html) to select the values to copy inside the JSON.
+
+It is suggested to avoid using the default CSV delimiter (` + "`,`" + `) as it could cause clashes with the JSON format. 
+Change the CSV delimiter with the ` + "`field_separator`" + ` configuration of the ` + "`[csv]`" + ` section. For instance:
+
+	[csv]
+		field_separator=";"
+
+### Example
+
+A possible filter configuration is:
+
+	[[filter]]
+	name="ExpandJSON"
+		[filter.config]
+		Source = "json_data"
+		[filter.config.Fields]
+		jfield1  = "field1"
+		jfield2  = "field2"
+
+The filter will transform the following input in the corresponding output:
+
+**Input:**
+
+| field1 | field2 |              json_data                 |
+| :----: | :----: | :------------------------------------: |
+|        |        | ` + "`{\"jfield1\":\"value1\", \"jfield2\":\"value2\"}`" + ` |
```

Comment:
It's a good idea to give an example for a filter which is somewhat more involved.
Maybe using a markdown table here is a bit overkill?
Just showing the TOML, what contains `json_data` and explaining that the filter outcome is to write `value1` into `field1` and `value2` into `field2` might be enough.

Reply:
Also, we don't need backslashes to represent a valid json string. Also I think we could drop the double quotes around `json_data` content and just write `| {"jfield1":"value1", "jfield2":"value2"} |`